Automattic/wp-calypso
# Atomic Downgrade: Adjust "Proceed with caution" wording

Fixes https://github.com/Automattic/wp-calypso/issues/56989

#### Changes proposed in this Pull Request

Changes the copy displayed on the "Proceed with caution" step prior cancelling a plan from an Atomic site, so users have a better idea of what's going to happen.

<img width="500" alt="Screen Shot 2021-10-14 at 16 42 53" src="https://user-images.githubusercontent.com/1233880/137340730-9be7edfd-a17e-4905-a799-0b31446be266.png">

#### Testing instructions

- Use the Calypso live link below and append `?flags=atomic/automated-revert` at the end of the URL.
- Switch to an Atomic site.
- Go to Upgrades > Purchases.
- Start the plan deactivation flow.
- Make sure the "Proceed with caution" contains the updated wording.
- Make sure there is a tooltip which displays extra detailed information.
- Make sure the atomic transfer date is correct and it's displayed in the expected format according to your locale.
